pkg/image/validation/kubernetes.go

```diff
@@ -44,19 +46,53 @@ func isKubernetesDefined(k8s *image.Kubernetes) bool {
 	return k8s.Version != ""
 }
 
-func validateNodes(k8s *image.Kubernetes) []FailedValidation {
+func validateNetwork(k8s *image.Kubernetes) []FailedValidation {
 	var failures []FailedValidation
+	var vipAddress = k8s.Network.APIVIP
+
+	if vipAddress == "" {
+		numNodes := len(k8s.Nodes)
+		// For single node clusters apiVIP is optional, test only multi-node ones
+		if numNodes > 1 {
+			failures = append(failures, FailedValidation{
+				UserMessage: "The 'apiVIP' field is required in the 'network' section when defining entries under 'nodes'.",
+			})
+		}
 
-	numNodes := len(k8s.Nodes)
-	if numNodes <= 1 {
-		// Single node cluster, node configurations are not required
 		return failures
 	}
 
-	if k8s.Network.APIVIP == "" {
+	ip, err := netip.ParseAddr(vipAddress)
+	if err != nil {
+		msg := fmt.Sprintf("Invalid cluster API address ('%s')", strings.ToLower(vipAddress))
 		failures = append(failures, FailedValidation{
-			UserMessage: "The 'apiVIP' field is required in the 'network' section when defining entries under 'nodes'.",
+			UserMessage: msg,
 		})
+
+		zap.S().Error(msg)
+
+		return failures
+	}
+
+	if !ip.IsGlobalUnicast() {
+		msg := fmt.Sprintf("Invalid non-unicast cluster API address (%s)", ip.String())
+		failures = append(failures, FailedValidation{
+			UserMessage: msg,
+		})
+
+		zap.S().Error(msg)
```

Comment:
Here we don't have an actual error value but this message is going to be logged both to the console, as well as the log file so this statement is not necessary.